apm-agent-plugins/apm-jdbc-plugin/src/main/java/co/elastic/apm/jdbc/PreparedStatementInstrumentation.java

```diff
@@ -62,7 +62,7 @@
     @Advice.OnMethodEnter
     public static Span onBeforeExecute(@Advice.This PreparedStatement statement) throws SQLException {
         if (tracer != null && jdbcHelper != null) {
-            final String sql = ConnectionInstrumentation.getSqlForStatement(statement);
+            final @Nullable String sql = ConnectionInstrumentation.getSqlForStatement(statement);
```

Comment:
I think the `@Nullable` here is not necessary

Reply:
It's not, but I thought it is a good hint/directive for anyone looking at this code